cirq-core/cirq/transformers/analytical_decompositions/two_qubit_to_fsim_test.py

```diff
@@ -83,7 +83,7 @@ def test_decompose_two_qubit_interaction_into_two_b_gates(obj: Any):
     desired_unitary = obj if isinstance(obj, np.ndarray) else cirq.unitary(obj)
     for operation in circuit.all_operations():
         assert len(operation.qubits) < 2 or operation.gate == _B
-    assert cirq.approx_eq(cirq.unitary(circuit), desired_unitary, atol=1e-6)
+    np.testing.assert_allclose(cirq.unitary(circuit), desired_unitary, atol=3e-5)
```

Comment:
Should this also get the "lose a lot of precision" notice?

Reply:
added